src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.TryGet.cs

```diff
@@ -39,6 +39,29 @@ public string GetString()
             return JsonReaderHelper.TranscodeHelper(span);
         }
 
+        /// <summary>
+        /// Reads value of the next JSON token for a comment from the source transcoded as a <see cref="string"/>,
+        /// preserving the comment delimiters.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">
+        /// Thrown if trying to get the value of the JSON token that is not a comment.
+        /// <seealso cref="TokenType" />
+        /// It will also throw when the JSON string contains invalid UTF-8 bytes, or invalid UTF-16 surrogates.
+        /// </exception>
+        public string GetComment()
+        {
+            if (TokenType != JsonTokenType.Comment)
+            {
+                throw ThrowHelper.GetInvalidOperationException_ExpectedComment(TokenType);
+            }
+            ReadOnlySpan<byte> span = HasValueSequence ? ValueSequence.ToArray() : ValueSpan;
+            while (span.LastIndexOfAny(JsonConstants.CarriageReturn, JsonConstants.LineFeed) == span.Length - 1)
```

Comment:
This is to handle the `\r\n` case in which two characters `\r` and `\n` are to be chomped off from the end. Since the search is in reverse direction, I thought `LastIndexOfAny` would eagerly match `\n` and hence would have to repeat for `\r`. Hence the loop.

Reply:
With changes to the `ConsumeComment` functionality, the above loop is no more required. Removed it.